test/unit/DetectChanges/BCBreak/FunctionBased/ParameterTypeContravarianceChangedTest.php

```diff
@@ -126,6 +133,14 @@ static function changed1(int $a, int $b) {}
        function changed2(int $a, int $b) {}
    }
 }
+
+namespace N5 {
+    interface A extends B {}
+    interface B {}
```

Comment:
The naming in this test makes things a bit complex to follow.

Let's rename `A` and `B` to make the sorting more clear instead.

`B` used in both examples as "child interface".